### ISSUE TITLE
Add a built product?

### ISSUE THREAD

**Issue by Jasonlhy** [google/xi-mac#82]
May be just add a built product to let people download and try?

**Comment by AllanLRH** [comment]
Would love for it to be available as a `brew cask install xi-mac`.

**Comment by cmyr** [comment]
We're still a ways away from offering a prebuilt package, although it's definitely an eventual goal.

**Comment by davelab6** [comment]
Would love to see this :)

**Comment by foisonocean** [comment]
any updates?

**Comment by cmyr** [comment]
We won't be offering a built product for a while yet, because it creates the impression that the software is ready to use, regardless of how vocally that is disclaimed. Although it might be frustrating for some users, having a slightly more difficult install process is probably useful for the time being. If you're really interested in trying out the editor and can't figure out the build instructions, I'd be happy to help.

**Comment by foisonocean** [comment]
@cmyr The problem for me is that I'm still using macOS 10.11, which is only allowed to install XCode 8, but xi-mac requires XCode 9, so I can't build it by myself. So I wonder if you could provide a built version, I can easily try the latest xi-mac.

**Comment by cmyr** [comment]
@foisonocean I have _zero_ confidence this will work, but you can try this:

It isn't a proper archive, just my build. You may need to have some libs installed, particularly oniguruma, available through homebrew. This also won't generate logs for errors originating in xi-core (#136) so you might want to open the app bundle and try running `xi-core` directly to debug?

https://cmyr.net/upload/XiEditor.zip

**Comment by foisonocean** [comment]
@cmyr This works for me on 10.11

**Comment by kylegoetz** [comment]
@cmyr works for me on 10.12 as well. Thanks. I really wanted to check this out, but Xcode takes up eleven billion zettabytes and my laptop has like 3GB of free space.

**Comment by yschimke** [comment]
Is the way forward here a brew recipe to build it?

**Comment by jansol** [comment]
It's a step in the right direction yes. However, as far as I know homebrew does not accept formulae for things that don't have proper versioned releases so it won't help much yet.

**Comment by yschimke** [comment]
You can host your own brew repo e.g. https://github.com/yschimke/homebrew-tap

works like `brew install yschimke/tap/okurl` -> okurl.rb

**Comment by jansol** [comment]
Trust issues aside, do those get any kind of binary caching? If not, that'd defeat the point of not having to install Xcode & rust.

**Comment by yschimke** [comment]
Yep - you could setit up so default downloads a binary you self host on github releases.  and --master builds from sources, so it autoupdates with builds.  But at least your build and getting build dependencies would also be automated.

But nothing like caching of built binaries by arbitrary users as a free build farm.

**Comment by jab** [comment]
Just got curious to try Xi enough to find my way to this issue, and noticed the most recent comment is a few years old. In case you're able to say, do you think providing e.g. a `brew install`able binary is still a few years away?

**Comment by jansol** [comment]
There won't be such a thing as the project [has effectively been shut down](https://github.com/xi-editor/xi-editor/issues/1298#issuecomment-717993371) after reaching its research goals. The core components are being used by some frontend projects as mentioned in the linked comment. Besides those, one of the recently more popular projects is [lapce](https://github.com/lapce/lapce).
